zebra/zebra_netns_notify.c

```diff
@@ -41,6 +41,10 @@
 #define ZEBRA_NS_POLLING_INTERVAL_MSEC     1000
 #define ZEBRA_NS_POLLING_MAX_RETRIES  200
 
+#if !defined(__GLIBC__)
```

Comment:
Isn't it enough to define it once (lib/zebra.h)?

Reply:
I had prior patch lingering, removed that and pushed again